bin/cli-helpers.js

```diff
@@ -7,9 +7,26 @@ function loadVerifyExamplesConfig(configPath) {
   */
 
   var verifyExamples = null;
+  var elmJson = null;
 
   try {
     verifyExamples = require(configPath);
+    if (verifyExamples["elm-root"] === undefined) {
+      throw new Error(
+        `elm-verify-examples.json at ${configPath} does not have an elm-root key`
```

Comment:
Maybe you are not doing this on purpose, but the default could be `..`, which is probably very commonly what is set anyway? And `elm-root` would be an optional override.

Reply:
Sounds like this would be a nice step to making the configuration file optional, which I think would be preferable.

Reply:
done!